make/config.mk

```diff
@@ -80,6 +80,9 @@ ENABLE_CUDA_RTC = 1
 # whether use CuDNN R3 library
 USE_CUDNN = 0
 
+# whether to use NVTX when profiling
+USE_NVTX = 0
```

Comment:
@KellenSunderland do you recommend enabling this in pip?

Reply:
I did measurements with a few models and didn't see any performance deltas.  I think it would be safe to enable in pip.

Reply:
Yes, as long as you are not running under nvprof nvtx calls are basically noops.